### ISSUE TITLE
Remove OxyPlot.Pdf

### ISSUE THREAD

**Issue by tibel** [oxyplot/oxyplot#866]
Don't maintain two PDF exporter implementations:
- one in core (NET40, PCL)
- one in OxyPlot.Pdf (NET40, NET45 and SL5)

**Comment by objorke** [comment]
I agree it would be great to get rid of the exporter based on PdfSharp. But currently the portable pdf exporter is lacking some functionality (font encoding, image formats)...

**Comment by GeertvanHorrik** [comment]
Ping me if this needs to be done for 2.0, then I'll remove this.

**Comment by nzain** [comment]
Found this issue while seaching for font encoding. I realized that the core `PortableDocument` currently only supports `FontEncoding.WinAnsiEncoding` and the method `PortableDocument.EncodeString(..)` does nothing. `PdfWriter` internally converts all strings to UTF8 when writing a file. A very short look at some [PDF spec](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf) (pg 85,86) suggests UTF16 (bigendian) shall be used, but eventually I didn't really understand it.

I'm familiar with text encoding, rather new to PDF creation - and willing to contribute. @objorke  can you give me a rough idea what you think needs to be implemented for unicode support? My personal motivation: create unicode characters like ≤ (\u2264) and the like, not be restricted to terrible old ANSI. Right now my time is very limited, but I might pick this up in the future, iff I know what to do :)

**Comment by objorke** [comment]
@nzain It would be great if you could contribute on this!
- this issue is about removing the OxyPlot.Pdf package, do not use this issue for the font issue
- #85 and #955 may be related, but we may want to create a separate issue for the font encoding topic
